app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -883,3 +886,13 @@ export class Dispatcher {
     }
   }
 }
+
+/**
+ * Joins filepath to the repository's base path, and normalizes path separators to the platform default.
+ */
+function joinAndNormalizePath(repository: Repository, filepath: string): string {
+  const fullPath = Path.join(repository.path, filepath)
+  // because Windows uses different path separators here
+  const normalized = Path.normalize(fullPath)
```

Comment:
@niik gosh, I'm trying to recall but I really can't. Maybe this was something that came from the Node 6.x days that is no longer needed?